README.md

```diff
@@ -1,39 +1,68 @@
 
-### Streamlit
+## Table of Contents
 
-#### D.Tomin
-[![](https://docs.streamlit.io/logo.svg)App file](streamlit_app.py)  
-[![](https://docs.streamlit.io/logo.svg)Demo app](https://articleprocessing.streamlit.app) (some features are disabled)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
 
-https://github.com/MyEvilpumpkin/ArticleProcessing/assets/24656713/ad4bf0ce-3e95-48a9-8f3d-ee4f62aa7f83
+## Features
 
-#### A.Oleynik
-**Model:** [papluca/xlm-roberta-base-language-detection](https://huggingface.co/papluca/xlm-roberta-base-language-detection)  
-**Run local:** streamlit run streamlit_app.py  
-**Details:** Type the required text in the input field and press Ctrl+Enter.  
-The expected language of the entered text will appear below the input field.
+- **Genre Definition**: Determines the genre of a given text. Uses the "MoritzLaurer/mDeBERTa-v3-base-mnli-xnli" model for zero-shot classification.
+- **Sentiment Analysis**: Analyzes the sentiment of a given text. Uses the "blanchefort/rubert-base-cased-sentiment" model.
+- **Language Detection**: Detects the language of a given text. Uses the "papluca/xlm-roberta-base-language-detection" model.
+- **Translation**: Translates a given text from one language to another. Uses the "Helsinki-NLP/opus-mt-ru-en" model.
+- **Summarization**: Summarizes a given text. Uses the "d0rj/rut5-base-summ" model.
 
-![View of the working application](https://github.com/MyEvilpumpkin/ArticleProcessing/assets/13471304/8a4bfc0b-d6c1-48ce-977c-c26417b18556)
 
-### FastAPI
+## Installation
 
-[![](!https://fastapi.tiangolo.com/ru/img/icon-white.svg)App file](web_app.py)
+You can set up and run ArticleProcessing in two ways: by installing the dependencies and running it directly on your machine, or by building a Docker image and running it as a Docker container.
 
-https://github.com/MyEvilpumpkin/ArticleProcessing/assets/24656713/6d9f6047-a72e-43bd-9fbc-77e6a4733d2e
+### Option 1: Install dependencies
 
-https://github.com/MyEvilpumpkin/ArticleProcessing/assets/24656713/3241c7f4-acd2-4169-a604-2fc10c736158
+1. Clone the repository:
 
-https://github.com/MyEvilpumpkin/ArticleProcessing/assets/24656713/dd59326c-a48b-47ca-a9ad-7effe988213f
+`git clone https://github.com/MyEvilpumpkin/ArticleProcessing.git`.
 
+2. Install the dependencies:
+
+`bash pip install -r requirements.txt`
+
+### Option 2: Use Docker
+
+1. Clone the repository:
+
+`git clone https://github.com/MyEvilpumpkin/ArticleProcessing.git`.
+
+2. Build the Docker image:
+
+`bash docker build -t articleprocessing .`
+
+## Usage
+
+To run the Streamlit app:
+
+`bash streamlit run streamlit_app.py`
+
+To run the FastAPI app:
+
+`bash uvicorn web_app:app --reload`
+
+To run the Docker container:
+
+`bash docker run -p 8501:8501`
```

Comment:
It's best not to write bash at the beginning of the command, since the choice of shell for these commands is not that important.  
Also pay attention to the "." after commands on lines 30 and 40, it can be confusing

Reply:
Regarding the docker commands, this is not entirely correct, but I’ll change it later when I’m done with the docker-compose file

Reply:
Removed shell name from commands, got rid of excess dots